variables.tf

```diff
@@ -67,14 +67,14 @@ variable "cos_plan" {
 }
 
 variable "cos_tags" {
-  description = "Optional list of tags to be added to cloud object storage instance. Only used if 'create_cos_instance' it true."
+  description = "A list of tags to apply to data in the Object Storage instance."
```

Comment:
Data is not tagged. Just the resource instance

Reply:
removed `data in`